tests/21-large-file-derivatives.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+set -e
+
+BASE_TEST_FOLDER="$(pwd)/$(dirname $0)/$(basename $0 .sh)"
+TESTCAFE_TESTS_FOLDER="${BASE_TEST_FOLDER}/testcafe"
+TESTCAFE_IMAGE=testcafe/testcafe
+
+# Start the backend that serves the media files to be migrated
+# Listens internally on port 80 (addressed as http://<assets_container>/assets/)
+startMigrationAssetsContainer
```

Comment:
On my system (which appears to be aster than yours or GH action's), there is a race condition, where file migration start running _before_ the migration assets container is up and running.  The net result is curl errors in the migration messages for image migrations, indicating an error connecting to `http://migration-assets...`.  

A simple `sleep 30` inserted after `startMigrationAssetsContainer` "fixed" the issue for me.  You can just add that, or add some sort curl wait loop.

Reply:
A curl loop would be more robust, for sure

Reply:
To close this loop - so to speak - curl is now invoked to check for the liveness of the assets container; `startMigrationAssetsContainer` will block until the container is up, or error out.